src/Sentry/Protocol/Envelopes/EnvelopeItem.cs

```diff
@@ -7,46 +7,28 @@
 using Sentry.Internal;
 using Sentry.Internal.Extensions;
 
-namespace Sentry.Protocol
+namespace Sentry.Protocol.Envelopes
 {
-    /// <summary>
-    /// Envelope item.
-    /// </summary>
-    public class EnvelopeItem : IDisposable, ISerializable
+    internal class EnvelopeItem : ISerializable, IDisposable
```

Comment:
If (when?) we end up making envelopes public again we'll need to add all these xml docs back.

Why remove it?